versions/3.0.4.md

```diff
@@ -3821,9 +3818,9 @@ Two examples of this:
 
 ## Security Considerations
 
-### OpenAPI Document Formats
+### OpenAPI Description Formats
 
-OpenAPI description documents use JSON, YAML, and JSON Schema, and therefore share their security considerations:
+OpenAPI Descriptions use JSON, YAML, and JSON Schema, and therefore share their security considerations:
```

Comment:
This would imply that all documents of a multi-document OAD MUST use the same format.

Which we can't require in a patch release.

```suggestion
OpenAPI documents use JSON, YAML, and JSON Schema, and therefore share their security considerations:
```

Reply:
> This would imply that all documents of a multi-document OAD MUST use the same format.

I don't see how that is the case -- can you explain why you think so?

In any event I think a different fix should be made. This PR eliminates all occurrences of "OpenAPI document" so we should not reintroduce one here.

Reply:
Ah, I read this as "use either JSON or YAML", but the addition of "and JSON Schema" hints that this is a combination. How about
```suggestion
OpenAPI Descriptions use a combination of JSON, YAML, and JSON Schema, and therefore share their security considerations:
```